piwik/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/continuouspipe/php7-apache
+
+MAINTAINER Alex Biddle <alex.biddle+cp-dockerfiles@inviqa.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y php-geoip php-dev libgeoip-dev && apt-get clean
+
+RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN echo "extension=geoip.so" >> /etc/php/7.0/apache2/php.ini
```

Comment:
A confd template that writes to `/etc/php/7.0/apache2/conf.d/geoip.ini` would be a more user friendly way to enable the geoip module, as users who use this image could control that template as a whole.